src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -20,6 +20,49 @@
     >
       <div v-html="t('data-planes.routes.items.intro', {}, { defaultMessage: '' })" />
       <KCard>
+        <search>
+          <FilterBar
+            class="data-plane-proxy-filter"
+            :placeholder="`service:backend`"
+            :query="route.params.s"
+            :fields="{
+              name: { description: 'filter by name or parts of a name' },
+              protocol: { description: 'filter by “kuma.io/protocol” value' },
+              service: { description: 'filter by “kuma.io/service” value' },
+              tag: { description: 'filter by tags (e.g. “tag: version:2”)' },
+              ...(can('use zones') && { zone: { description: 'filter by “kuma.io/zone” value' } }),
+            }"
+            @change="(e) => route.update({
+              ...Object.fromEntries(e.entries()) as Record<string, string | undefined>,
+            })"
+          />
+
+          <XSelect
+            label="Type"
+            :selected="route.params.dataplaneType"
+            @change="(value: string) => route.update({ dataplaneType: value })"
+          >
+            <template #selected="{ item }: { item: 'all' | 'standard' | 'builtin' | 'delegated'}">
+              <XIcon
+                v-if="item !== 'all'"
+                :size="KUI_ICON_SIZE_40"
+                :name="item"
+              />
+              {{ t(`data-planes.type.${item}`) }}
+            </template>
+            <template
+              v-for="item in (['all', 'standard', 'builtin', 'delegated'] as const)"
+              :key="item"
+              #[`${item}-option`]
+            >
+              <XIcon
+                v-if="item !== 'all'"
+                :name="item"
+              />
+              {{ t(`data-planes.type.${item}`) }}
+            </template>
+          </XSelect>
+        </search>
         <DataLoader
```

Comment:
For others: this is identical to https://github.com/kumahq/kuma-gui/pull/3089/files?w=1#diff-9d4c2f48081cf9b206eb208a8f15ad06ede0411df7c550dbd938e1926d89b2d9L23-L62